# type
string FULL_MAP = full_map
string VIRTUAL_WALL_MAP = virtual_wall_map

string name
string type
float32 resolution
uint32 width
uint32 height
float64 origin_x
float64 origin_y
uint8[] data
